src/main/java/core/mate/academy/service/BulldozerProducer.java

```diff
@@ -0,0 +1,24 @@
+package core.mate.academy.service;
+
+import core.mate.academy.model.Bulldozer;
+import java.util.List;
+
+public class BulldozerProducer implements MachineProducer<Bulldozer> {
+
+    @Override
+    public List<Bulldozer> get() {
+        return List.of(
+                new Bulldozer("bulldozerBig",
+                        "BigBob",
+                        "Black",
+                        3),
+                new Bulldozer("bulldozerMedium",
+                        "DickRay",
+                        "Pink",
+                        2),
+                new Bulldozer("bulldozerMini",
+                        "TinyMate",
+                        "Yellow",
+                        1));
```

Comment:
I'd recommend squeezing it. I mean starting each parameter from the new line is unnecessary. Please fix the same issue in all places